ci/Jenkinsfiles/release.groovy

```diff
@@ -160,18 +160,17 @@ pipeline {
       steps {
         container('maven') {
           script {
-            def jiraIssueFetchers = [
-                type                 : 'jira',
-                jql                  : "project = ${JIRA_PROJECT} and fixVersion = ${JIRA_MOVING_VERSION}",
-                newJiraVersion       : [
-                    project    : env.JIRA_PROJECT,
-                    name       : env.JIRA_RELEASED_VERSION,
-                    releaseDate: LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE),
-                    released   : true,
-                ],
-                jiraMovingVersionName: env.JIRA_MOVING_VERSION,
-            ]
-            nxProject.release(issuesFetchers: [jiraIssueFetchers], tagPrefix: 'release-')
+            nxProject.release(
+              jql                  : "project = ${JIRA_PROJECT} and fixVersion = ${JIRA_MOVING_VERSION}",
+              newJiraVersion       : [
+                project    : env.JIRA_PROJECT,
+                name       : env.JIRA_RELEASED_VERSION,
+                releaseDate: LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE),
+                released   : true,
+              ],
+              jiraMovingVersionName: env.JIRA_MOVING_VERSION,
+              tagPrefix: 'release-',
```

Comment:
I have a small doubt about this args's position

Reply:
It is at the right position, note that it was already a method parameter.

You can find "top-level" arguments there: 
https://github.com/nuxeo/platform-ci-shared-library/blob/3dc1f46817b20194e687ca0c175978bbe3c16d78/src/org/nuxeo/platform/Project.groovy#L55-L65